tests/integration/test_charm.py

```diff
@@ -125,7 +130,10 @@ async def test_upgrade(self, ops_test: OpsTest):
         await ops_test.model.integrate(f"{ISTIO_PILOT}:ingress", f"{APP_NAME}:ingress")
         await ops_test.model.integrate(f"{ISTIO_PILOT}:ingress-auth", f"{APP_NAME}:ingress-auth")
         await ops_test.model.integrate(f"{APP_NAME}:oidc-client", f"{DEX_AUTH}:oidc-client")
-        await ops_test.model.applications[APP_NAME].set_config({"public-url": PUBLIC_URL})
+
+        # TODO: remove after releasing ckf-1.9/stable, this has been preserved to avoid breaking
+        # integration tests.
+        await ops_test.model.applications[APP_NAME].set_config({"public-url": "http://foo.io"})
```

Comment:
how do we need this since there's no code handling this config option?

Reply:
If we keep it though, you could add a reminder (or have an issue) to remove this once we release.

Reply:
This test case is testing the upgrade between ckf-1.8 (still has the config option) and latest/edge (option is removed). If I did not keep it like this, ckf-1.8 will be blocked all the time w/o this config, preventing the upgrade.

I will add an issue for removing it.

Reply:
Fine, let's add it in kubeflow 1.9 epic I guess or top of backlog. If you do this, feel free to resolve this comment

Reply:
Here is the issue https://github.com/canonical/oidc-gatekeeper-operator/issues/167